core/examples/luigi-example-react/README.md

```diff
@@ -1,38 +1,29 @@
 # Sample Luigi application written in ReactJS
 
-## Overview
+> :information_source: This is an **example app only** aimed to give an insight on the setup with Luigi Microfronteds all within a React App. Make sure to adjust the settings according to your development and production requirements.  
 
-This is the ReactJS-based sample application which runs with Luigi framework.
+This is a simple **minimalistic** React Based App, serving **Luigi Core** on the default **index.html** and some simple React based microfrontends through **sampleapp.html**.
 
+Since usually you would serve **Core** and **Client microfrontends** in different locations/environments this example only serves as a playground and starting point to understand the setup. 
```

Comment:
```suggestion
This is a minimalistic React-based app which serves **Luigi Core** on the default `index.html` and some simple React-based micro frontends through `sampleapp.html`.

Since you would usually serve **Core** and **Client** micro frontends in different locations/environments, this example only serves as a playground and starting point to understand the setup. 
```